cypress.Dockerfile

```diff
@@ -1,15 +1,15 @@
 # # # # # # # # # # # # # # # # # # # # 
 # Docker image with Cypress dependencies
-# 
+#
 # RUN steps: 
-# 	
-# 	install cypress, copy config files, run
-# 
+#
+#   install cypress, copy config files, run
+#
 # # # # # # # # # # # # # # # # # # # # 
 FROM cypress/included:7.5.0
 COPY package.json .
 RUN npm install
 
 COPY cypress cypress
 COPY cypress.json .
-CMD ["npx","cypress","run","--headless","--browser", "chrome","--record","--key","d88e9357-758a-4bad-9485-8cf21b7e1643"]
\ No newline at end of file
+CMD ["npx","cypress","run","--headless","--browser", "chrome", "--record","--key", "d88e9357-758a-4bad-9485-8cf21b7e1643", "--parallel", "--ci-build-id", "$COMMIT_INFO_SHA"]
```

Comment:
It looks like you're passing a record key in as a variable above, but you're still using it hardcoded here. Should this line use the variable?

Reply:
cannot make that variable work. so this PR stick to add more machines to speed up the cypress run time.